pkgs/development/tools/misc/stlink/default.nix

```
@@ -1,17 +1,19 @@
-{ stdenv, fetchurl, cmake, libusb1 }:
+{ stdenv, fetchFromGitHub, cmake, libusb1 }:
 
 # IMPORTANT: You need permissions to access the stlink usb devices. 
 # Add services.udev.pkgs = [ pkgs.stlink ] to your configuration.nix
 
 let
-  version = "1.3.0";
+  version = "1.5.1";
 in
 stdenv.mkDerivation {
```

should be able to get rid the `let ... in ..` as well
```suggestion
stdenv.mkDerivation rec {
```